docs/docsite/Makefile

```
@@ -64,6 +64,10 @@ coredocs: core_htmldocs
 generate_rst: collections_meta config cli keywords plugins testing
 core_generate_rst: collections_meta config cli keywords base_plugins testing
 
+# At the moment localizing the plugins and collections is not required for the ongoing
+# localisation effort. It will come at a later time.
+gettext_generate_rst: config cli keywords testing
```

```suggestion
gettext_generate_rst: collections_meta config cli keywords testing
```
collections_meta is information about the metadata in collections so it is in scope for this.